pkg/route/api/validation/validation.go

```diff
@@ -85,5 +85,25 @@ func validateTLS(route *routeapi.Route) fielderrors.ValidationErrorList {
 		msg := fmt.Sprintf("invalid value for termination, acceptable values are %s, %s, %s, or emtpy (no tls specified)", routeapi.TLSTerminationEdge, routeapi.TLSTerminationPassthrough, routeapi.TLSTerminationReencrypt)
 		result = append(result, fielderrors.NewFieldInvalid("termination", tls.Termination, msg))
 	}
+	result = append(result, validateNoDoubleEscapes(tls)...)
 	return result
 }
+
+// validateNoDoubleEscapes ensures double escaped newlines are not in the certificates.  Double
+// escaped newlines may be a remnant of old code which used to replace them for the user unnecessarily.
+func validateNoDoubleEscapes(tls *routeapi.TLSConfig) fielderrors.ValidationErrorList {
+	allErrs := fielderrors.ValidationErrorList{}
+	if strings.Contains(tls.CACertificate, "\\n") {
+		allErrs = append(allErrs, fielderrors.NewFieldInvalid("caCertificate", tls.CACertificate, "double escaped new lines (\\\\n) are invalid"))
```

Comment:
for sanity, use raw string literals anywhere you find yourself double escaping backslashes:

\`double escaped new lines (\\\\n) are invalid\`

(I had to do horrible things to get that comment to appear the way I meant it to)